README.md

```diff
@@ -1 +1,124 @@
-# guru-discord-bot
\ No newline at end of file
+<h1 align="center">Guru Discord Bot</h1>
+<div align="center">
+    <strong>
+        Szerveren lévő jogosultságok/skillek kezelő botja a <a href="https://discord.gg/A5yC5Hhxrs">Step into Community</a> Discord szerveren.
+    </strong>
+</div>
+<br/>
+<div align="center">
+
+[![Discord](https://img.shields.io/discord/848942109191962665.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/A5yC5Hhxrs)
+
+
+</div>
+<br/>
+
+## Funkcionalitás
+
+- Csatlakozó felhasználók üdvözlése, ami után reakciók segítségével tudják magukhoz rendelni a szerepköröket
+- A bot által nyújtott parancsok/súgó lekérése (`/skills`)
+- Szerepkörök/skillek listázása a szerveren (`/list-skills`)
+- Szerepkörök felhaszálóhoz rendelése (`/i-know`)
+- Szerepkörök törlése/visszautasítása (`/i-dont-know`)
+
+## Jellemzők
+
+- **Nincs perzisztencia** - A bot nem tartja számon, hogy melyik felhasználó melyik szerepkörrel rendelkezik, ezek lekérésére a Discord API biztosít metódusokat, de jövőben elképzelhető, hogy egy perzisztens réteg is be lesz iktatva.
+- **Slash parancsok natív támogatásának hiánya** - A Javacord könyvtár !!egyelőre!! nem támogatja a natív `slash-command`-okat, de az új verziók valamelyikében elképzelhető, hogy elérhető lesz, emiatt a parancsok nem jelenekk meg egy `/` billentyű leütése után.
+
+## Fordítás és futtatás
+
+A projekt [Quarkus](https://quarkus.io) keretrendszerre épül és a [Javacord](https://javacord.org/) könyvtárat használja a Discord API által nyújtott funkciók kihasználására.
+
+A fordításhoz a következőkre van szükség:
+
+  * Java 11 vagy afölötti verzió.
+  * Maven 3.6.x (A projekt tartalmaz egy wrapped Maven-t, használható az is)
+
+Ezután a tényleges fordítás (JAR összeállítása):
+
+~~~~
+$ ./mvnw package
```

Comment:
It is also a mistake from my side, but can you please remove the `$` signs from the scripts? Since GitHub now provides a copy button in each code block, it is more of an annoyance now than something that actually helps.